src/OpenTracing/OpenTracing.csproj

```diff
@@ -1,11 +1,15 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard1.0</TargetFramework>
+    <TargetFrameworks>netstandard1.0;net45</TargetFrameworks>
     <Description>This library is a .NET implementation of the OpenTracing API. To fully understand this platform API, it's helpful to be familiar with the OpenTracing project and terminology more generally.
 
 For the time being, mild backwards-incompatible changes may be made without changing the major version number. As OpenTracing and opentracing-csharp mature, backwards compatibility will become more of a priority.</Description>
     <PackageTags>opentracing;distributed-tracing;tracing;logging</PackageTags>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(NonWindowsBuild)' == 'true' ">
```

Comment:
can't this just be `'$(OS)' == 'Windows'` since msbuild injects the variable and then you don't have to pass in the proper cli args

Reply:
Thanks ! I didn't know this variable existed, it is simpler that way indeed.